setup/context/commands.go

```diff
@@ -5,14 +5,15 @@ import (
 	"crypto/tls"
 	"encoding/json"
 	"fmt"
-	"github.com/quintilesims/layer0/common/aws/provider"
-	"github.com/quintilesims/layer0/common/aws/s3"
-	"github.com/quintilesims/layer0/common/config"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"strings"
 	"time"
+
+	"github.com/quintilesims/layer0/common/aws/provider"
```

Comment:
[nitpick] You're probably using `goimports` rather than `gofmt`, which creates this line break between stdlib packages and 3rd party libraries. Our styleguide thus far has been to use `gofmt`, which does not add this line break.